Ciac-16436 - iZOOlogic automation (#44410)
demisto/content: Packs/iZOOlogic/README.md

## What does this pack do?

> Fetches and manages threat incidents from iZOOlogic, enabling automated ingestion, incident creation, and advanced filtering for brand protection and threat management.

Reviewer: ```suggestion
This content pack fetches and manages incidents from iZOOlogic, enabling automated ingestion, issue creation, and advanced filtering for brand protection and threat management.
```
@dtroushinsky please confirm:
- alert object names in iZOOlogic (incidents or threat incidents?)
- should incidents be changed to issues when referring to the objects in Cortex?

Author: @julieschwartz18  The alert object name is incidents and it should remain incidents (and not issues)

This pack includes an integration that connects to the iZOOlogic API to:

> - **Fetch incidents** — Automatically ingest threat incidents into Cortex for centralized security monitoring, correlation, and threat analysis.

Reviewer: ```suggestion
- **Fetch incidents**: Automatically ingest incidents into Cortex for centralized security monitoring, correlation, and threat analysis.
```

> - **Create incidents** — Report new security incidents (such as phishing, brand abuse, and malware) directly to iZOOlogic for investigation.

Reviewer: ```suggestion
- **Create incidents**: Report new security incidents (such as phishing, brand abuse, and malware) directly to iZOOlogic for investigation.
```

@dtroushinsky incidents or issues?

Author: @julieschwartz18 The terminology that came from the vendor is incidents.
"This API method is used to report new security incidents for investigation. Creates a new reported incident case that enters the review workflow."

> - **Search incidents** — Query incidents with advanced filters including date range, incident type, threat level, and brand.

Reviewer: ```suggestion
- **Search incidents**: Query incidents with advanced filters including date range, incident type, threat level, and brand.
```
